src/media_listener.js

```diff
@@ -23,6 +24,8 @@ MediaListener.prototype._buildMediaQueryLists = function(mediaQueries) {
 };
 
 MediaListener.prototype._setupListeners = function(name, mediaQuery) {
+  if (!window.matchMedia) return
+
```

Comment:
From the README:

> You can also optionally specify a `defaultMediaQuery` prop, which will get used when no other media queries match. This can also be useful if you're rendering your component server side.

Will the `defaultMediaQuery` value still be passed through to the child component if `matchMedia` is missing?